src/CONST.ts

```diff
@@ -5263,6 +5279,11 @@ const CONST = {
             REPORT_ID: 'reportID',
             KEYWORD: 'keyword',
         },
+        QUERY_KIND: {
+            CANNED_QUERY: 'cannedQuery',
+            CUSTOM_QUERY: 'customQuery',
+        },
+        DEFAULT_QUERY: 'type:all sortBy:date sortOrder:desc',
```

Comment:
```suggestion
        DEFAULT_QUERY: 'type:expense status:all sortBy:date sortOrder:desc offset:0',
```

Reply:
That being said, I think the parser returns the default case already

Reply:
Yeah good catch 😄 I've removed it